debian/control

```diff
@@ -0,0 +1,15 @@
+Source: neofs-node
+Section: neofs
```

Comment:
Section should be changed
https://www.debian.org/doc/debian-policy/ch-archive.html#s-subsections
https://packages.debian.org/unstable/

Reply:
updated to "misc"